### DISCUSSION TITLE
How can I set additional releaseRules of the action?

### DISCUSSION THREAD

**Discussion by pawellabaj** [semantic-release-extras/next-release-version#24] [Q&A]
I have dependabot commits in my repo that looks as follow:
```
build: maven: (deps): bump junit-jupiter.version from 5.10.0 to 5.10.1 (#136)

Bumps `junit-jupiter.version` from 5.10.0 to 5.10.1.
(...)
```

I want to make them to bump patch number in my release version.

I tried to set following rule with `.releaserc.yml` or `package.json` files

```
[
        "@semantic-release/commit-analyzer",
        {
          "preset": "angular",
          "releaseRules": [
            {
              "type": "build",
              "scope": "deps",
              "release": "patch"
            }
          ]
        }
      ]
```

It looks that he action is not looking for configuration overriding in these files.

**Is it possible to define additional ruleset? How to do this?**

**Comment by EricCrosson** [comment]
Hi @pawellabaj, that sounds a bit outside the scope of this repository. You probably have already seen this, but I believe these docs contain the most relevant information around configuring commit-analyzer rules: https://github.com/semantic-release/commit-analyzer

I'm not too familiar with custom rules, but in your case, I wonder if the `"scope": "deps"` is what's causing your rule to not match the example commit message. In my head, I'd figure `{"type": "build", "scope": "deps"}` to match a commit message like:

```
build(deps): lorem ipsum
```

Have you tried removing `scope` from your release rule?

**Reply by pawellabaj** [reply]
I tried different entries.

It looks that the action uses always the same package.json:
```
Run : install semantic-release package manifest
  : install semantic-release package manifest
  mkdir -p "$node_workdir"
  cp "/home/runner/work/_actions/semantic-release-action/next-release-version/v4/internal/package.json" "/home/runner/work/_actions/semantic-release-action/next-release-version/v4/internal/package-lock.json" "$node_workdir/"
```

**Reply by EricCrosson** [reply]
Oh yeah, you're right. It does appear that the current implementation is fundamentally incapable of supporting release rules:

https://github.com/semantic-release-action/next-release-version/blob/dcf34f066c62e7c8b3b04ce6aaf10dfe54bc6f4b/internal/action.yml#L60-L67
